### PR TITLE
I added an offline method for cIOS installation.

### PR DESCRIPTION
**Pull request by Snare-Hawk**
yeah
also i formatted it with prettier so that changed something but it looks prettier now probably

### CODE REVIEW

**Comment by Peter0x44** [comment]
Great! I've wanted this for probably over a year now, I was just too lazy to sit down and do it to completion.
Will review it now.

File: _pages/en_US/cios.md [hunk 1]
```diff
@@ -16,61 +16,90 @@ If you have a Wii mini, install [this cIOS](cios-mini) instead. Attempting to in
...
 ##### Section I - Downloading
 
 1. Download d2x cIOS Installer and extract it to your SD card or USB drive.
 1. Insert your SD card or USB drive into your Wii, and launch d2x cIOS Installer from the Homebrew Channel.
+</div>
+<div id="without-connection" class="blanktabcontent" markdown="1">
+
+##### Section I - Downloading
+
+1. Download, extract, and open [NUS Downloader](https://github.com/WiiDatabase/nusdownloader/releases/download/v1.9-mod-nusfix/NUSD-Mod-NUS-Fix.zip).
```
Comment (Peter0x44) [inline _pages/en_US/cios.md:42]: I think this should just be "run" instead of "open"

File: _pages/en_US/cios.md [hunk 2]
```diff
@@ -16,61 +16,90 @@ If you have a Wii mini, install [this cIOS](cios-mini) instead. Attempting to in
...
+1. Select "Database", "IOS", and then "IOS56" and select the "Latest Version".
```
Comment (Peter0x44) [inline _pages/en_US/cios.md:43]: "and" is written twice in this sentence. remove it, and use an "oxford comma" too

File: _pages/en_US/cios.md [hunk 3]
```diff
@@ -16,61 +16,90 @@ If you have a Wii mini, install [this cIOS](cios-mini) instead. Attempting to in
...
+   - Ensure that "Pack WAD" is checked and "Patch IOS" is unchecked.
+1. Repeat the previous step for "IOS57" and "IOS38".
+1. Once you have downloaded all three cIOSes, there will be a folder named `titles` in the same folder as the NUS Downloader. Open the folder and navigate through until you find the three WAD files you downloaded. Place each of them on the root of your SD card.
+1. Download d2x cIOS Installer and extract it to your SD card or USB drive.
```
Comment (Peter0x44) [inline _pages/en_US/cios.md:47]: I think we should say "extract it to the apps folder on your SD card or USB drive"
Though, I noticed this poor wording is on the priiloader page too, and then it also mentions a bullet point that doesn't make sense
I will fix this for the priiloader page later.

**Comment by Snare-Hawk** [comment]
all right cool i'll update with these changes, thanks!
~~if i know how to use github correctly, i'll probably be fine~~

**Comment by Peter0x44** [comment]
LGTM

**Pull request merged**